php/libraries/NDB_Page.class.inc

```diff
@@ -243,14 +243,15 @@ class NDB_Page
      * Adds a hidden element to the current page. Note if the hidden element
      * needs a value it should be added to the defaults.
      *
-     * @param string $name  The name of the hidden element to add
-     * @param array  $value The value of the hidden element to add
+     * @param string $name       The name of the hidden element to add
+     * @param array  $value      The value of the hidden element to add
+     * @param array  $attributes Additional html attributes for element
      *
      * @return none
      */
-    function addHidden($name, $value=null)
+    function addHidden($name, $value=null, $attributes=null)
     {
-        $this->form->addElement('hidden', $name, $value);
+        $this->form->addElement('hidden', $name, $value, $attributes);
```

Comment:
Just doing this won't work, you need to make appropriate changes to LorisForm to allow for attributes to be added to hidden elements. Look at the functions `addElement`, `addHidden` and `hiddenHtml` to reflect these changes